### PR TITLE
Feature/arm spec

### PR DESCRIPTION
**Pull request by mexanick**
Add ARM spec template

### CODE REVIEW

File: mfRPMRules.mk [hunk 1]
```diff
@@ -59,24 +59,41 @@ endif
...
 	sed -i 's#__package__#$(Package)#' $(PackagePath)/rpm/$(PackageName).spec
 	sed -i 's#__packagename__#$(PackageName)#' $(PackagePath)/rpm/$(PackageName).spec
 	sed -i 's#__version__#$(PACKAGE_VER_MAJOR).$(PACKAGE_VER_MINOR).$(PACKAGE_VER_PATCH)#' $(PackagePath)/rpm/$(PackageName).spec
 	sed -i 's#__release__#$(PACKAGE_FULL_RELEASE)#' $(PackagePath)/rpm/$(PackageName).spec
-	sed -i 's#__prefix__#$(INSTALL_PREFIX)#' $(PackagePath)/rpm/$(PackageName).spec
+    ifeq ($(Arch),arm)
+	    sed -i 's#__prefix__#/mnt/persistent/gemdaq#' $(PackagePath)/rpm/$(PackageName).spec
+    else
+	    sed -i 's#__prefix__#$(INSTALL_PREFIX)#' $(PackagePath)/rpm/$(PackageName).spec
+    endif
```
Comment (jsturdy) [inline mfRPMRules.mk:96]: I think this block should be replaced with L95 only, and the `Makefile` should appropriately set the `INSTALL_PREFIX`

File: specTemplateARM.spec [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+Packager: %{_author}
+#BuildArch: %{_buildarch}
+License: __license_
+# Group: Applications/extern
+URL: %{_url}
+BuildRoot: %{_tmppath}/%{_packagename}-%{_version}-%{_release}-buildroot
+Prefix: %{_prefix}
+Requires: __requires_list__
+BuildRequires: __build_requires_list__
+AutoReqProv: no
```
Comment (jsturdy) [inline specTemplateARM.spec:36]: Probably *only* need `AutoReq: no` here

**Review by jsturdy — COMMENTED** [review]
I *think* we can get away without a separate spec template
* We can wrap all `%pre`/`%post`/`%build`/`%prep` targets inside an `if %not_arm` block, a la:
```
%if %not_arm
%pre

%prep

# %setup
## requires Source tag
%build
%endif
```
* Similarly, we can add the following to the appropriate RPM targets
```
if %is_arm
AutoReq: no
%endif
```
* We can also adapt the `rpmbuild` command to check if `ARCH` is `arm` like so:
```
RPM_OPTIONS=
ifeq ($(ARCH),arm)
        RPM_OPTIONS=--define "_binary_payload 1"
endif
...

.PHONY: makerpm
makerpm:
        mkdir -p $(PackagePath)/rpm/RPMBUILD/{RPMS/$(ARCH),SPECS,BUILD,SOURCES,SRPMS}
        rpmbuild --quiet -ba -bl \
        --define "_requires $(REQUIRES_LIST)" \
        --define "_build_requires $(BUILD_REQUIRES_LIST)" \
        --define  "_topdir $(PWD)/rpm/RPMBUILD" $(PackagePath)/rpm/reedmuller.spec \
        $(RPM_OPTIONS) --target "$(ARCH)"
        find  $(PackagePath)/rpm/RPMBUILD -name "*.rpm" -exec mv {} $(PackagePath)/rpm \;
```

File: specTemplateARM.spec (hunk 2 above)
Comment (mexanick) [reply specTemplateARM.spec:36]: no, we need the other lines to handle our own dependencies

**Comment by mexanick** [comment]
I will close this PR and provide a new one with a single modified template

**Pull request closed without merging**